packages/slice-machine/src/modules/pushChangesSaga/index.ts

```diff
@@ -1,191 +0,0 @@
-  "PUSH_CHANGES.REQUEST",
-  "PUSH_CHANGES.RESPONSE",
-  "PUSH_CHANGES.FAILURE",
-)<ChangesPushSagaPayload, undefined, Limit | InvalidCustomTypeResponse>();
-
-export const sortDocumentLimits = (limit: Readonly<Limit>) => ({
-  ...limit,
-  details: {
-    ...limit.details,
-    customTypes: [...limit.details.customTypes].sort(
-      (doc1, doc2) => doc2.numberOfDocuments - doc1.numberOfDocuments,
-    ),
-  },
-});
-
-const MODAL_KEY_MAP = {
-  ["INVALID_CUSTOM_TYPES"]: ModalKeysEnum.REFERENCES_MISSING_DRAWER,
-  ["SOFT"]: ModalKeysEnum.SOFT_DELETE_DOCUMENTS_DRAWER,
-  ["HARD"]: ModalKeysEnum.HARD_DELETE_DOCUMENTS_DRAWER,
-};
-
-export function* changesPushSaga({
-  payload,
-}: ReturnType<typeof changesPushCreator.request>): Generator {
-  const startTime = Date.now();
-  const { changedSlices, changedCustomTypes } = payload;
-
-  const sliceChanges = changedSlices.map((sliceChange) => ({
-    id: sliceChange.slice.model.id,
-    type: "Slice" as const,
-    libraryID: sliceChange.slice.from,
-    status: sliceChange.status,
-  }));
-  const customTypeChanges = changedCustomTypes.map((customTypeChange) => ({
-    id: customTypeChange.customType.id,
-    type: "CustomType" as const,
-    status: customTypeChange.status,
-  }));
-
-  // Creating a new payload with the correct format
-  const pushPayload = {
-    confirmDeleteDocuments: payload.confirmDeleteDocuments,
-    changes: [...sliceChanges, ...customTypeChanges],
-  };
-
-  try {
-    const response = (yield call(pushChanges, pushPayload)) as SagaReturnType<
-      typeof pushChanges
-    >;
-
-    if (response) {
-      // sending failure event
-      yield put(changesPushCreator.failure(sortDocumentLimits(response)));
-      // Tracking when a limit has been reached
-      void telemetry.track({
-        event: "changes:limit-reach",
-        limitType: response.type,
-      });
-      // Open the corresponding drawer
-      yield put(
-        modalOpenCreator({
-          modalKey: MODAL_KEY_MAP[response.type],
-        }),
-      );
-      return;
-    }
-
-    const serverState = (yield call(getState)) as SagaReturnType<
-      typeof getState
-    >;
-    yield put(
-      refreshStateCreator({
-        env: serverState.env,
-        remoteCustomTypes: serverState.remoteCustomTypes,
-        localCustomTypes: serverState.customTypes,
-        libraries: serverState.libraries,
-        remoteSlices: serverState.remoteSlices,
-        clientError: serverState.clientError,
-      }),
-    );
-
-    // Tracking the success of the push
-    void trackPushChangesSuccess({ ...payload, startTime });
-
-    // Send global success event
-    yield put(changesPushCreator.success());
-
-    // Display success toaster
-    yield put(
-      openToasterCreator({
-        content: "All slices and types have been pushed",
-        type: ToasterType.SUCCESS,
-      }),
-    );
-  } catch (error) {
-    if (isUnauthenticatedError(error) || isUnauthorizedError(error)) {
```

Comment:
I removed this support in the new code. 

First, it was not working, and even if you were unauthorized it was displaying the classic error. And also because you cannot see the push button without being logged in anyway. WDYT?